python/tests/test_slicebyvalue.py

```diff
@@ -79,18 +79,18 @@ def test(sliceable):
         test(self._d)
 
     def test_assign_variable_to_range_dataarray(self):
-        self._d['a']['x', 1.5 * sc.units.dimensionless:4.5 *
-                     sc.units.dimensionless].data = sc.Variable(
-                         dims=['x'], values=[6.0, 6.0, 6.0], unit=sc.units.m)
-
-        assert self._d['a'].data.values.tolist() == [1.0, 6.0, 6.0, 6.0, 1.4]
+        with pytest.raises(sc.DataArrayError):  # readonly
```

Comment:
Since there is no alternative way one could modify these values in place (can now only make a copy), I suggest this test (and the ones below) should have a new name that suggests that it is supposed to throw.

Reply:
renamed